server/auth/auth_test.go

```diff
@@ -54,37 +54,9 @@ func setupTestHelper(t *testing.T) *TestHelper {
 
 func TestGetSession(t *testing.T) {
```

Comment:
There is nothing in this test left now, so we can delete this.

Reply:
Removed it.